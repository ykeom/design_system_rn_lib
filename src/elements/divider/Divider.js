import React, { Component } from 'react';
import { View, StyleSheet, Text } from 'react-native';

const styles = StyleSheet.create({
  root: {},
});

class Divider extends Component {
  render() {
    return (
      <View style={styles.root}>
        <Text>divider</Text>
      </View>
    );
  }
}

export default Divider;
