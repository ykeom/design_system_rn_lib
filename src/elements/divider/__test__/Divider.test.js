import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Divider from '../Divider';

describe('Divider', () => {
  it('should render', () => {
    const component = shallow(<Divider />);

    expect(component.length).toBe(1);
    expect(toJson(component)).toMatchSnapshot();
  });
});
