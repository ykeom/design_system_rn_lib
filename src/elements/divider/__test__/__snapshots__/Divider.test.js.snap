// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Divider should render 1`] = `
<View
  style={Object {}}
>
  <Text>
    divider
  </Text>
</View>
`;
